src/conventions/h-group/clue-interpretation/interpret-clue.js

```diff
@@ -297,6 +302,32 @@ export function interpret_clue(game, action) {
 			}
 		}
 
+		// If there's a valid connection that doesn't require a bluff, a bluff is not a valid interpretation.
+		const no_bluff_connections = all_connections.filter(connection => connection.connections.length > 0 && !connection.connections[0].bluff);
```

Comment:
If we don't need `no_bluff_connections`, we can just use `.some()` instead of `.filter()` so that it'll exit early once it finds one.